site/content/installation/installing-ngf/manifests.md

```diff
@@ -19,6 +19,12 @@ To complete this guide, you'll need to install:
 
 Deploying NGINX Gateway Fabric with Kubernetes manifests takes only a few steps. With manifests, you can configure your deployment exactly how you want. Manifests also make it easy to replicate deployments across environments or clusters, ensuring consistency.
 
+- If you’d like to use NGINX Plus:
+  - To pull from the F5 Container registry, configure a docker registry secret using your JWT token from the MyF5 portal by following the instructions from [here](https://docs.nginx.com/nginx-gateway-fabric/installation/ngf-images/jwt-token-docker-secret). Make sure to specify the secret using `nginxGateway.serviceAccount.imagePullSecretName` or `nginxGateway.serviceAccount.imagePullSecretsNames` parameter.
+  - Alternatively, pull an Nginx Gateway Fabric image with NGINX Plus and push it to your private registry by following the instructions from [here]({{ ref "installation/ngf-images/pulling-ngf-image.md" }}).
+  - Alternatively, you can build an Nginx Gateway Fabric image with NGINX Plus and push it to your private registry by following the instructions from [here]({{< relref "installation/building-the-images.md" >}}).
+  - Update the `nginxGateway.image.repository` field of the `values-plus.yaml` accordingly.
+
```

Comment:
These configuration options don't apply to manifests, only helm. We should add steps to the `make generate-manifests` target to also generate manifests for an NGINX Plus installation. NIC has manifests for both OSS and Plus, so we should too.

Reply:
as part of this PR?

Reply:
Yes, it should be in the ticket. Our docs will reference manifests to install, so they should exist either before or while we add those docs.

Reply:
okay will do so

Reply:
updated makefile to generate plus manifests